TreeView: add the "getting started" article (#2591)
DevExpress/devextreme-documentation: concepts/05 UI Components/TreeView/00 Getting Started with TreeView/00 Getting Started with TreeView.md

> The TreeView component displays textual data from a local or remote source in a hierarchical form.

Reviewer: Why did you write the adjective "textual"? If by "data" you mean only text, then just write "text". If by "data" you mean literally anything, including text, then just leave the word "data".

> The TreeView component represents textual data from a local or remote source in the form of a single-column tree view. The component allows users to select and view loaded data.

Reviewer 2: ```suggestion
The TreeView component displays textual data from a local or remote source in a hierarchical form.
```

> This tutorial shows how to create and configure the TreeView. As a result, you will get a UI component that looks as follows:

Reviewer 3: ```suggestion
This tutorial shows how to create and configure the TreeView. The image below shows the resulting UI:
```

> This tutorial shows how to add the TreeView to a page, bind it to data, and configure its core features. As a result, you will get a UI component that looks as follows:

Reviewer: > This tutorial shows how to add the TreeView to **a page**, bind **it** to data, and configure **its** core features.

Let's take a closer look at this sentence. The pronoun "it" points at "a page", not at "the TreeView" as you most likely meant originally.

> <div class="simulator-desktop-container" data-view="/Content/Applications/GettingStartedWith/TreeView/index.html, /Content/Applications/GettingStartedWith/TreeView/index.js, /Content/Applications/GettingStartedWith/TreeView/index.css"></div>

Reviewer 2: The links produce error 404. They need doc versions.

> Refer to the following sections for details on each configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-treeview" target="_blank">Getting Started with DevExtreme TreeView</a>.

Reviewer 3: ```suggestion
Refer to the following sections for information on each configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-treeview" target="_blank">Getting Started with DevExtreme TreeView</a>.
```

Reviewer 3: You use **following** a lot in your text

> Refer to the following sections for details on each configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-treeview" target="_blank">getting-started-with-treeview</a>.

Reviewer 2: ```suggestion
Refer to the following sections for details on each configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-treeview" target="_blank">Getting Started with DevExtreme TreeView</a>.
```
